examples/flow.cpp

```diff
@@ -16,12 +18,40 @@
   You should have received a copy of the GNU General Public License
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
+
+
+#if HAVE_CONFIG_H
 #include "config.h"
+#endif // HAVE_CONFIG_H
+
+
+#include <dune/common/version.hh>
+
+#include <opm/core/utility/platform_dependent/disable_warnings.h>
+
+#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 3)
+#include <dune/common/parallel/mpihelper.hh>
+#else
+#include <dune/common/mpihelper.hh>
+#endif
+
+#if HAVE_DUNE_CORNERPOINT && defined(WANT_DUNE_CORNERPOINTGRID)
+#define USE_DUNE_CORNERPOINTGRID
```

Comment:
I suggest we define this macro (and WANT_...) to 1, then we may use #if or #ifdef to query it. I would also prefer using #if since that is what is used in most other places now (at the very least in this file).